fastapitableau/statics/css/styles.css

```diff
@@ -1,76 +1,520 @@
+}
+
+.md-typeset h1 {
+    margin: 0 0 1.25emline-height:1.3;
+    font-size: 36px;
+}
+
+.md-typeset h1,
+.md-typeset h2 {
+    letter-spacing: -.01em
+}
+
+.md-typeset h2 {
+    margin: 1.6em 0 .64em;
+    font-size: 1.5625em;
+    line-height: 1.4
+}
+
+.md-typeset h3 {
+    margin: 1.6em 0 .8em;
+    font-weight: bold;
+    font-size: 1.25em;
+    line-height: 1.5;
+    letter-spacing: -.01em
+}
+
+.md-typeset h4 {
+    margin: 1em 0;
+    font-weight: 700;
+    letter-spacing: -.01em
+}
+
+.md-typeset a {
+    color: var(--md-typeset-a-color);
+    word-break: break-word
+}
+
+.md-typeset a,
+.md-typeset a:before {
+    transition: color 125ms
+}
+
+.md-typeset a:focus,
+.md-typeset a:hover {
+    color: var(--md-accent-fg-color)
+}
+
+.md-typeset ul {
+    list-style-type: disc
+}
+
+.md-typeset ol,
+.md-typeset ul {
+    margin-left: .625em;
+    padding: 0
+}
+
+.md-typeset ol li,
+.md-typeset ul li {
+    margin-bottom: .5em;
+    margin-left: 1.25em
+}
+
+.md-typeset ol li:last-child,
+.md-typeset ul li:last-child {
+    margin-bottom: 0
+}
+
+.md-typeset img,
+.md-typeset svg {
+    max-width: 100%;
+    height: auto
+}
+
+html {
+    height: 100%;
+    overflow-x: hidden;
+    font-size: 125%
+}
+
+body {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    min-height: 100%;
+    font-size: .5rem;
+    background-color: var(--md-default-bg-color)
+}
+
+/* .md-grid {
+    max-width: 61rem;
+    margin-right: auto;
+    margin-left: auto
+} */
+
+.md-container {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1
+}
+
+.md-main {
+    flex-grow: 1
+}
+
+.md-main__inner {
+    display: flex;
+    height: 100%;
+}
+
+.md-ellipsis {
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis
+}
+
+:root {
+    --md-clipboard-icon: url("data:image/svg+xml;charset=utf-8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'><path d='M19 21H8V7h11m0-2H8a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h11a2 2 0 0 0 2-2V7a2 2 0 0 0-2-2m-3-4H4a2 2 0 0 0-2 2v14h2V3h12V1z'/></svg>")
+}
+
+.md-content {
+    flex-grow: 1;
+    overflow: hidden;
+    scroll-padding-top: 51.2rem
+}
+
+.md-content__inner {
+    margin: 0 .8rem 1.2rem 20px;
+    padding-top: .6rem;
+    max-width: 710px
+}
+
+.md-content__inner:before {
+    display: block;
+    height: .4rem;
+    content: ""
+}
+
+.md-content__inner>:last-child {
+    margin-bottom: 0
+}
+
+.md-header {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: 2;
+    color: var(--md-primary-bg-color);
+    background-color: var(--md-primary-fg-color);
+    box-shadow: 0 0 .2rem transparent, 0 .2rem .4rem transparent
 }
-.route {
-  margin-bottom: 3rem;
-  max-width: 1000px;
+
+.md-header[data-md-state=shadow] {
+    box-shadow: 0 0 .2rem rgba(0, 0, 0, .1), 0 .2rem .4rem rgba(0, 0, 0, .2);
+    transition: transform .25s cubic-bezier(.1, .7, .1, 1), box-shadow .25s
 }
 
-.route h3 {
```

Comment:
This is the missing style referenced above

Reply:
There was also a block in `home.css` that applied to `pre` and `code`, and we were using `code` in the User Guide. This is just from memory for later reference for me